File: bubble_sort.js

function bubbleSort(arr){
    len = arr.length
    for(i=0;i<len;i++){
        for(j=0;j<(len-i-1);j++){
            if(arr[j]>arr[j+1]){
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
            }
        }
    }
    console.log(arr)
}

var arr = [234, 43, 55, 63, 5, 6, 235, 547];
bubbleSort(arr);

File: bubblesort.py
def bubblesort(arr):
    for i in range(len(arr)):
        for j in range(len(arr)-i-1):
            if arr[j]>arr[j+1]:
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
    return arr
    
arr = [234, 43, 55, 63, 5, 6, 235, 547]
print(bubblesort(arr))